examples/build_all.sh

```diff
@@ -6,8 +6,12 @@ echo $(rustc --version)
 pushd $(dirname ${BASH_SOURCE[0]})
 
 for d in "status-message"  $(ls -d */ | grep -v -e "status-message\/$"); do
-    echo building $d;
-    (cd "$d"; ./build.sh);
+    for directory in $(find $d -type d); do
+        if [ -d "$directory/src" ]; then
+            echo building $d;
+            (cd "$d"; cargo near build non-reproducible-wasm; cp target/wasm32-unknown-unknown/release/*.wasm res/;);
```

Comment:
unfortunately, the output paths are incorrect. 
i suggest just removing all `res/` paths as it's not clear now, what they could be used for, 

and removing all `cp` commands too.